modules/update-service-graph-data.adoc

```diff
@@ -5,13 +5,13 @@
 [id="update-service-graph-data_{context}"]
 = Creating the OpenShift Update Service graph data container image
 
-The OpenShift Update Service requires a graph data container image, from which the OpenShift Update Service retrieves information about channel membership and blocked update edges. Graph data is typically fetched directly from the update graph data repository. In environments where an internet connection is unavailable, loading this information from an init container is another way to make the graph data available to the OpenShift Update Service. The role of the init container is to provide a local copy of the graph data, and during pod initialization, the init container copies the data to a volume that is accessible by the service.
-
 [NOTE]
 ====
 The oc-mirror OpenShift CLI (`oc`) plugin creates this graph data container image in addition to mirroring release images. If you used the oc-mirror plugin to mirror your release images, you can skip this procedure.
 ====
```

Comment:
I basically shifted this note about graph data images being unnecessary with oc-mirror higher up to the top of the section. 

But if anyone thinks this entire module should be deleted so that our "updating with OSUS" page only documents the oc-mirror + OSUS workflow, let me know.